2023/report.md

```diff
@@ -0,0 +1,155 @@
+    - Monorepo: Reduced overhead and improved developer experience
+- What should be improved:
+    - Diversity: No non-male participants this time
+    - Scope: The delivery goals were ambitious and task complexity hard to predict
+    - Cohesion: The effort got diffuse towards the end of the year, not all work got done; more check-ins needed
+    - Nixpkgs tooling and documentation: Still a major source of friction
+    - Efficiency: Mob programming is not a silver bullet; we have to figure out a sensible balance which tasks to do alone or in a group, and when
+
+Overall, and as predicted, better preparation greatly reduced the challenges reported from the 2021 and 2022 programs — such as clear objectives and task assignments — while keeping some freedom for participants to follow their tech interests.
+This came at the cost of a smaller number of software projects that were packaged in this period.
+
+## Lessons learned
+
+Many insights from the 2021 and 2022 editions were implemented successfully, and prior improvements could be reproduced.
+Some things have not been addressed though, but should be in the next iteration:
+
+- Improve the technical introduction so participants can get started more quickly.
+  For example, provide pointers to documentation on key aspects of the ecosystem, or re-introduce the mentor role.
+- Keep up motivation until the end.
+  A final presentation or closing ceremony could help.
+  Participants should get NixCon 2024 tickets for an opportunity to meet developers in person.
+  This could be combined with a hiring event.
+  This time we had two participants present at a Nix workshop, which was well-received by the audience.
+- Consider more support roles, such as technical writer, community manager, web master.
+  This time we focused exclusively on software development.
+  In 2022 there were alternative tracks, but those did not see much interest, possibly because they were announced after the application phase.
+  But its clear that eventually, if we want to improve the overall user and developer experience, we will need dedicated work done in various problem domains.
+  This is also a chance to increase diversity among participants.
+- Plan for additional outreach efforts.
+  Make time for participants to write experience reports, technical articles, or other content to convey what we’re doing and why.
+  This could also be implemented as a follow-up stipend.
+- Make Summer of Nix more than just about NGI.
+  Asking companies to submit and fund project proposals could help to leverage the visibility of the program, and to make progress on long-standing issues that can’t be addressed through the purpose-bound grants.
+  It could also be coupled with a hiring event to help companies with talent-acquisition.
+
+We also learned some new lessons:
+
+It’s vital to have at least one person to be available for answering questions and making more high-level decisions.
```

Comment:
This is more of a "context known and understood" rather than "authority granted" issue, so again, a question of knowledge transfer. It's hard in principle to make responsible forward-looking decisions without knowing why things are the way they are. And it turns out that writing down all the details properly takes so much time that it has to be balanced with other ongoing tasks. My suggestion to have someone with broad context available is based on such a trade-off.